### PR TITLE
add meta data to cargo.toml

### PR DESCRIPTION
**Pull request by pwang7**
This is the Rust version of the [K8S host path CSI](https://github.com/kubernetes-csi/csi-driver-host-path) demo.

### CODE REVIEW

File: csi/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        let res = executor::block_on(resp).expect("failed to get GetPluginInfoResponse");
+        println!("GetPluginInfo result: {:?}", res);
+        assert_eq!(res.1.name, "io.datenlord.csi.test_plugin");
+        assert_eq!(res.1.vendor_version, "0.1.0");
+
+        let resp = client
+            .get_plugin_capabilities(RequestOptions::new(), GetPluginCapabilitiesRequest::new())
+            .join_metadata_result();
+        let res = executor::block_on(resp).expect("failed to get GetPluginCapabilitiesResponse");
+        println!("GetPluginCapabilities result: {:?}", res);
```
Comment (rogercloud) [inline csi/src/main.rs:125]: assert?

File: csi/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+        let resp = client
+            .probe(RequestOptions::new(), ProbeRequest::new())
+            .join_metadata_result();
+        let res = executor::block_on(resp).expect("failed to get ProbeResponse");
+        println!("Probe result: {:?}", res);
```
Comment (rogercloud) [inline csi/src/main.rs:131]: assert?

File: csi/src/main.rs [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    use grpc::RequestOptions;
+    use std::fs;
+
+    #[test]
+    fn test_run_server() {
+        let socket_file = "/tmp/csi.sock";
+        if std::path::Path::new(socket_file).exists() {
+            fs::remove_file(socket_file).expect("failed to remove socket file");
+        }
+        std::thread::spawn(move || super::run_server(socket_file));
```
Comment (rogercloud) [inline csi/src/main.rs:100]: Join the thread before the test finished.

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -31,6 +31,7 @@ jobs:
     name: Check
     runs-on: ubuntu-latest
     steps:
+      - run: sudo apt install -y libprotobuf-dev protobuf-compiler
```
Comment (rogercloud) [inline .github/workflows/ci.yml:34]: Do we need to run "build" command to compile proto file?

Comment (pwang7) [reply .github/workflows/ci.yml:34]: `build.rs` will compile proto file.

**Review by rogercloud — APPROVED** [review]

**Pull request merged**